### PR TITLE
feat(#768): add ability to cancel tutorial

### PR DESCRIPTION
**Pull request by chao-mu**
No longer draft.

### CODE REVIEW

**Comment by jackstenglein** [comment]
The IconButton has a weird shape while hovering over it: Screenshot 2024-09-02 at 11 28 18 AM

File: frontend/src/tutorial/Tutorial.tsx [hunk 1]
```diff
@@ -58,16 +54,14 @@ const Tutorial: React.FC<TutorialProps> = ({ name, steps, zIndex }) => {
                         zIndex: zIndex || 100,
                     },
                 }}
-                disableCloseOnEsc
-                disableOverlayClose
```
Comment (jackstenglein) [inline frontend/src/tutorial/Tutorial.tsx:62]: Can we leave disableOverlayClose that was somebody doesn't accidentally close it?

File: frontend/src/tutorial/TutorialTooltip.tsx [hunk 2]
```diff
@@ -19,14 +20,24 @@ const TutorialTooltip: React.FC<TooltipRenderProps> = ({
...
             data-cy='tutorial-tooltip'
             sx={{ maxWidth: 'md', width: { xs: '95vw', sm: undefined } }}
             {...tooltipProps}
         >
-            <CardHeader title={step.title} />
+            <CardHeader
+                title={step.title}
+                action={
+                    <IconButton arra-label='close' onClick={closeProps.onClick}>
+                        <Typography component='span' color='Gray'>
```
Comment (jackstenglein) [inline frontend/src/tutorial/TutorialTooltip.tsx:35]: Removing the typography component here and just passing CloseIcon will probably fix the weird shape. You can pass a color directly to the close icon. Also, can we put gray in lowercase?

Comment (chao-mu) [reply frontend/src/tutorial/TutorialTooltip.tsx:35]: Sadly, the color prop type does not accept this value - unless my memory fails me, hence the typography. However, the weird shape (if you're talking about the one on hover) is there regardless. I will try again with these regards though, if I can't fix it with the methods you advise, what then?

File: frontend/src/tutorial/Tutorial.tsx (hunk 1 above)
Comment (chao-mu) [reply frontend/src/tutorial/Tutorial.tsx:62]: The first time I used it I kept clicking on the overlay trying to get it to close. It's the trained behavior of many apps that clicking away from the dialog closes it. Maybe this is a question for Post?

**Review by jackstenglein — APPROVED** [review]

**Pull request merged**